booking-module/server/index.js

```diff
@@ -39,35 +38,25 @@ app.get('/rooms/:listingId/booking', (req, res) => {
 
 app.post('/api/rooms/:listingId/booking', (req, res) => {
   const { body } = req;
+  body.listing_id = req.params.listingId;
   console.log(body);
-  booking.create({
-    id: req.params.listingId,
-    check_in: body.check_in,
-    check_out: body.check_out,
-    booking_location: body.booking_location,
-    user: body.user,
-  }, (err, data) => {
+  db.createBooking(body, (err, data) => {
     if (err) { return res.sendStatus(500); }
     return res.status(201).json(data);
   });
 });
 
 app.patch('/api/rooms/:listingId/booking', (req, res) => {
   const { body } = req;
-  booking.findByIdAndUpdate(body.bookingId, {
-    check_in: body.check_in,
-    check_out: body.check_out,
-    booking_location: body.booking_location,
-    user: body.user,
-  }, (err, data) => {
+  db.updateBooking(body, (err, data) => {
     if (err) { return res.sendStatus(500); }
     return res.status(209).json(data);
   });
 });
 
 app.delete('/api/rooms/:listingId/booking', (req, res) => {
```

Comment:
Is it necessary to have `:listingId` in your path and `id` in your request body?